src/main/kotlin/step3/racingcar/domain/CurrentTurn.kt

```diff
@@ -0,0 +1,20 @@
+package step3.racingcar.domain
+
+class CurrentTurn(var _currentTurn: Int = 0) {
```

Comment:
```suggestion
class CurrentTurn(var _currentTurn: Int = START_TURN) {
```
START_TURN 상수를 활용할 수 있을 것 같아요

Reply:
넵!